### PR TITLE
feat(HACBS-2273): use dependabot to update actions

### PR DESCRIPTION
**Pull request by Troy876**
Adds a new github actions workflow, dependabot.yml, to automatically check for specified dependency updates and opens a PR when an update is detected.

### CODE REVIEW

**Review by davidmogar — APPROVED** [review]

File: .github/workflows/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    reviewers:
+      - "davidmogar"
+      - "johnbieren"
+      - "theflockers"
+      - "scoheb"
+      - "happybhati"
+      - "mmalina"
+      - "kasemAlem"
+      - "Troy876"
+
```
Comment (davidmogar) [inline .github/workflows/dependabot.yml:41]: It still seems that there's something going on in the last line. Could you fix it?  The empty line is not needed, just the character.

**Review by davidmogar — CHANGES_REQUESTED** [review]
Rebase main as well so it can be merged (`git fetch && git rebase origin/master`).

File: .github/workflows/dependabot.yml (hunk 1 above)
Comment (Troy876) [reply .github/workflows/dependabot.yml]: I've removed the last line and I believe I've rebased main, everything should be fine now.

Comment (Troy876) [reply .github/workflows/dependabot.yml]: Fixed the empty line issue.

**Comment by mmalina** [comment]
I see this in the commit message:

> Signed-off-by: Troy <barrtroy02@gmail.com>

@Troy876 can you use your Red Hat email?

**Review by mmalina — APPROVED** [review]

**Review by davidmogar — APPROVED** [review]

**Review by kasemAlem — APPROVED** [review]

**Review by johnbieren — APPROVED** [review]

**Pull request merged**